### ISSUE TITLE
Add interface to install nugget/o8g back

### ISSUE THREAD

**Issue by brine** [octgn/OCTGN#2064]
Add interface to install nugget/o8g back

**Comment by kellyelton** [comment]
Doesn't need to be fancy. I know we were trying to discourage people from installing local games, but having people navigate to a local feed is probably too unintuitive in hindsight.

**Comment by brine** [comment]
yeah I think the code is already there?  Just needs a control in the games manager I bet
